Sources/DayView.swift

```diff
@@ -109,23 +144,44 @@ public class DayView: UIView, TimelinePagerViewDelegate {
   }
   
   private func configureLayout() {
-    if #available(iOS 11.0, *) {
       dayHeaderView.translatesAutoresizingMaskIntoConstraints = false
-      timelinePagerView.translatesAutoresizingMaskIntoConstraints = false
-      
-      dayHeaderView.leadingAnchor.constraint(equalTo: safeAreaLayoutGuide.leadingAnchor).isActive = true
-      dayHeaderView.trailingAnchor.constraint(equalTo: safeAreaLayoutGuide.trailingAnchor).isActive = true
-      dayHeaderView.topAnchor.constraint(equalTo: safeAreaLayoutGuide.topAnchor).isActive = true
+     
       let heightConstraint = dayHeaderView.heightAnchor.constraint(equalToConstant: headerHeight)
       heightConstraint.priority = .defaultLow
-      heightConstraint.isActive = true
       
-      timelinePagerView.leadingAnchor.constraint(equalTo: safeAreaLayoutGuide.leadingAnchor).isActive = true
-      timelinePagerView.trailingAnchor.constraint(equalTo: safeAreaLayoutGuide.trailingAnchor).isActive = true
-      timelinePagerView.topAnchor.constraint(equalTo: dayHeaderView.bottomAnchor).isActive = true
-      timelinePagerView.bottomAnchor.constraint(equalTo: bottomAnchor).isActive = true
-    }
+      NSLayoutConstraint.activate([
+        dayHeaderView.leadingAnchor.constraint(equalTo: safeAreaLayoutGuide.leadingAnchor),
+        dayHeaderView.trailingAnchor.constraint(equalTo: safeAreaLayoutGuide.trailingAnchor),
+        dayHeaderView.topAnchor.constraint(equalTo: safeAreaLayoutGuide.topAnchor),
+        heightConstraint
+      ])
   }
+    
+    // TODO: Create global constaints for tableView and set here
+    private func layoutTableView() {
+        guard let tableView = tableView else { return }
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            tableView.topAnchor.constraint(equalTo: dayHeaderView.bottomAnchor),
+            tableView.leadingAnchor.constraint(equalTo: safeAreaLayoutGuide.leadingAnchor, constant: horizontalSpacing),
+            tableView.trailingAnchor.constraint(equalTo: safeAreaLayoutGuide.trailingAnchor, constant: -horizontalSpacing),
+            tableView.bottomAnchor.constraint(equalTo: bottomAnchor)
+        ])
+    }
+    
+    private func layoutTimelinePagerView() {
+        timelinePagerView.translatesAutoresizingMaskIntoConstraints = false
+        timelinePagerView.leadingAnchor.constraint(equalTo: safeAreaLayoutGuide.leadingAnchor, constant: horizontalSpacing).isActive = true
+        timelinePagerView.trailingAnchor.constraint(equalTo: safeAreaLayoutGuide.trailingAnchor, constant: -horizontalSpacing).isActive = true
+        timelinePagerView.topAnchor.constraint(equalTo: dayHeaderView.bottomAnchor).isActive = true
+        timelinePagerView.bottomAnchor.constraint(equalTo: bottomAnchor).isActive = true
+    }
+    
+    public func switchView() {
```

Comment:
For me it is not clear hoe to switch mode - by calling switchView() method or by setting calendarMode property and handle change in didSet property observer.